python/pyspark/sql/readwriter.py

```diff
@@ -692,8 +734,7 @@ def orc(self, path, mode=None, partitionBy=None, compression=None):
                             This will override ``orc.compress``. If None is set, it uses the
                             default value, ``snappy``.
 
-        >>> orc_df = spark.read.orc('python/test_support/sql/orc_partitioned')
```

Comment:
@GregBowyer Could you exclude these changes from this PR? that could help to this PR to merged easily.